packages/sync-actions/test/product-sync-base.spec.js

```diff
@@ -13,6 +13,7 @@ describe('Exports', () => {
       'meta',
       'references',
       'prices',
+      'pricesCustom',
```

Comment:
I'm not sure if we really need a new action group here, we could have reused the existing `prices` action group here instead.

Reply:
yea true 👍 , i just saw it like this on category `export const actionGroups = ['base', 'references', 'meta', 'custom', 'assets']`, so i thought to separate it like category.
I also think it should be same as within price actions

Reply:
as we discussed, and as you told me :), i would be better to separate it with action, so we give flexibility choosing the actions

Reply:
What is an `action group` if I may ask? Sorry for derailing the conversation here.

Reply:
Don't be :), as i understood, `actionGroups` is a way to custom or default the running actions `doMapActions`, which then choose which actions should get generated ex: `updatePrice` which belong to `prices` action group

Reply:
Honestly until now, not sure which is better, but i think giving the option is better!